Uplift/Controllers/GymDetailViewController.swift

```diff
@@ -52,9 +53,11 @@ class GymDetailViewController: UIViewController {
     init(gym: Gym) {
         super.init(nibName: nil, bundle: nil)
         self.gymDetail = GymDetail(gym: gym)
-
-        for facility in gymDetail.facilities where facility.name == "Fitness Center" {
-            self.equipment = categorizeEquipment(equipmentList: facility.details[0].equipment)
+        for facility in gymDetail.facilities {
+            facilitiesDropdownCellStatuses.append(.closed)
+            if facility.name == "Fitness Center" {
+                self.equipment = categorizeEquipment(equipmentList: facility.details[0].equipment)
```

Comment:
we should have a check for `!facility.details.isEmpty` so that we don't crash on line 59